backend/director/llm/litellm.py

```
@@ -0,0 +1,159 @@
+                    )
+
+            return LLMResponse(
+                content=response.choices[0].message.content or "",
+                tool_calls=tool_calls,
+                finish_reason=response.choices[0].finish_reason,
+                send_tokens=getattr(usage, "prompt_tokens", 0) or 0,
+                recv_tokens=getattr(usage, "completion_tokens", 0) or 0,
+                total_tokens=getattr(usage, "total_tokens", 0) or 0,
+                status=LLMResponseStatus.SUCCESS,
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)